catalyst_voices/lib/pages/account/account_page.dart

```diff
@@ -0,0 +1,225 @@
+import 'package:catalyst_voices/widgets/buttons/voices_icon_button.dart';
+import 'package:catalyst_voices/widgets/buttons/voices_text_button.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+final class AccountPage extends StatelessWidget {
+  const AccountPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SingleChildScrollView(
+        child: Padding(
+          padding: const EdgeInsets.all(32),
+          child: Column(
+            children: [
+              _Header(),
+              Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 16),
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    const SizedBox(height: 32),
+                    _Tab(),
+                    const SizedBox(height: 48),
+                    _KeychainCard(
+                      connectedWallet: 'Lace',
+                      roles: const [
+                        'Voter (Default)',
+                        'Proposer',
+                        'Drep',
+                      ],
+                      onRemoveKeychain: () => debugPrint('Keychain removed'),
+                    ),
+                  ],
+                ),
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class _Header extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: 350,
+      child: Stack(
+        children: [
+          Container(
+            decoration: BoxDecoration(
+              image: DecorationImage(
+                image: CatalystImage.asset(
+                  VoicesAssets.images.accountBg.path,
+                ).image,
+                fit: BoxFit.cover,
+              ),
+            ),
+          ),
+          Positioned(
+            top: 24,
+            left: 8,
+            child: VoicesIconButton.filled(
+              onTap: () {
+                GoRouter.of(context).pop();
+              },
+              style: ButtonStyle(
+                backgroundColor: WidgetStateProperty.all(
+                  Theme.of(context).colors.elevationsOnSurfaceNeutralLv1White,
+                ),
+                foregroundColor: WidgetStateProperty.all(
+                  Theme.of(context).colors.iconsForeground,
+                ),
+              ),
+              child: VoicesAssets.icons.arrowNarrowLeft.buildIcon(),
```

Comment:
Can you use `LeftArrowButton` that we already have in `voices_buttons`? Maybe you will need to override theme

Reply:
`LeftArrowButton` is an icon button, but not filled, and we need here filled variant